### PR TITLE
interfaces/utf: Add Ledger to U2F devices

### PR DESCRIPTION
**Pull request by TamtamHero**
The Ledger products support both FIDO U2F and FIDO2 - please add it to the U2F device list.

For verification: 
http://www.linux-usb.org/usb.ids
https://github.com/Yubico/libu2f-host/blob/master/70-u2f.rules#L68-L69

Thanks

### CODE REVIEW

**Review by zyga — APPROVED** [review]
Looks good to me. Thank you!

**Review by anonymouse64 — DISMISSED** [review]
lgtm, thanks for providing the verification links!

File: interfaces/builtin/u2f_devices.go [hunk 1]
```diff
@@ -121,6 +121,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "0483",
 		ProductIDPattern: "a2ac",
 	},
+	{
+		Name:             "Ledger Blue + Nano S + Nano X",
+		VendorIDPattern:  "2c97",
+		ProductIDPattern: "0000|0001|0004",
```
Comment (jdstrand) [inline interfaces/builtin/u2f_devices.go:127]: https://github.com/Yubico/libu2f-host/blob/master/70-u2f.rules#L68-L69 doesn't list the Ledger Blue (0000), but http://www.linux-usb.org/usb.ids does. Googling, it seems the blue is a u2f device. Can someone confirm? Once confirmed, +1 to apply.

Comment (zyga) [reply interfaces/builtin/u2f_devices.go:127]: It seems to be https://github.com/LedgerHQ/app-u2f 

For the record: I also noticed that the referenced document doesn't list the blue product number but it looked sensible so I didn't ask about it.

Comment (TamtamHero) [reply interfaces/builtin/u2f_devices.go:127]: Ahem, I was not up to date with our latest firmware upgrade:
We used to have these values but we recently changed the way our devices announce their productID: https://www.ledger.com/windows-10-update-sunsetting-u2f-tunnel-transport-for-ledger-devices/
So I have to make a PR first on https://github.com/Yubico/libu2f-host (I will also add the Blue product there).
Meanwhile I'm updating the productIDs here with their new values. Sorry for the back-pedaling.

Comment (TamtamHero) [reply interfaces/builtin/u2f_devices.go]: PR waiting here: https://github.com/Yubico/libu2f-host/pull/151

Comment (zyga) [reply interfaces/builtin/u2f_devices.go]: Please clarify the product IDs as asked in the referenced PR. Right now it seems that this is wrong in one or other way.

Comment (TamtamHero) [reply interfaces/builtin/u2f_devices.go]: https://github.com/Yubico/libu2f-host/pull/151 has been updated and merged.
I just updated this one to match what I've done there (adding back old productIDs next to updated ones)

**Comment by mvo5** [comment]
@TamtamHero Thanks again for your PR. Could you please sign the contributor agreement https://ubuntu.com/legal/contributors and push the code with the email-adress that is used to sign the CLA? If we can help just let us know :)

**Comment by TamtamHero** [comment]
Here's where I land: https://launchpad.net/~tamtamhero

**Comment by chipaca** [comment]
@TamtamHero thank you!
@mvo5 with that, I can confirm lp:~tamtamhero has signed the CLA.

**Comment by chipaca** [comment]
@TamtamHero as @mvo5 said, you probably need to re-push your work using that email address for the automated checker to work :-/

**Comment by chipaca** [comment]
woo look at it go :-D

**Comment by TamtamHero** [comment]
Yay, it should be good now :D

**Review by mvo5 — APPROVED** [review]

**Comment by TamtamHero** [comment]
:wave: 
Any news ?

**Comment by TamtamHero** [comment]
hello-gif

**Comment by mvo5** [comment]
@TamtamHero Hey, thanks for the PR and sorry that it took so long to get merged.

**Review by anonymouse64 — COMMENTED** [review]
post merge LGTM, apologies for the delay here

**Pull request merged**